fingertip/plugins/cleanup.py

```diff
@@ -100,6 +101,45 @@ def tempfiles(older_than='4h', location=None):
                                       temp.AUTOREMOVE_PREFIX not in f))
 
 
+def saviour(expired_for='10d'):
+    if expired_for != 'all':
+        adjusted_time = time.time() - units.parse_time_interval(expired_for)
+    else:
+        adjusted_time = 0
+    for root, dirs, files in os.walk(path.SAVIOUR + '/_', topdown=False):
```

Comment:
`glob.glob('**/.last_mirrored', recursive=True, include_hidden=True)` might be a quicker and lighter-on-the-logs way.